python/ray/experimental/state.py

```diff
@@ -304,3 +304,33 @@ def client_table(self):
       node_info[node_ip_address].append(client_info_parsed)
 
     return node_info
+
+  def log_table(self):
+    """Fetch and return a dictionary of log file names to outputs, sorted by
+    IP address.
```

Comment:
quick style comment, the first line of the docstring should fit in one line (following the convention in http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).